### DISCUSSION TITLE
Trying to get an entity to attack me

### DISCUSSION THREAD

**Discussion by jason7141** [bridge-core/editor#628] [Q&A] [answered]
I'm trying to get an entity to attack myself when I get a certain bounty level using scoreboard. When it gets to the bounty level it adds a tag that works! I'm trying with a custom character 
And when I use minecraft:behavior_nearest_attackable_target, I'm using filter. Test: has_tag, value: warrant. it dosnt detect the player with the tag. Is their any other way to set this up? I need for my custom police

**Comment by jason7141** [comment, marked as answer]
Here's my code for police
![Screenshot_20220926-203517_Chrome](https://user-images.githubusercontent.com/105606159/192404995-b16cdd4d-0ed8-459a-9ff5-1e1344632a84.jpg)

**Reply by solvedDev** [reply]
I would try changing the subject from "player" to "other".

**Reply by jason7141** [reply]
OK ty I will give a try let ya know!

**Reply by jason7141** [reply]
Yes that worked ty!
